SSA/Projects/InstCombine/Alive.lean

```diff
@@ -2,7 +2,9 @@
 -- Auto generated alive statements
 -- Do not import these into the default tree, because they take a long
 -- time to elaborate and typecheck.
--- import SSA.Projects.InstCombine.AliveAutoGenerated
+-- Import this by default, since the file has a `#exit` that checks the first two
+--   theorems. This ensures that the file (and transitively, its dependencies) are not broken.
+import SSA.Projects.InstCombine.AliveAutoGenerated
```

Comment:
Can we make this part of a separate build facet, so that local builds don't have to include it, but we can run the whole file on CI?